A1.md

```diff
@@ -3,14 +3,14 @@ HTTP CONNECT Proxy Support
 * Author(s): Mark D. Roth (roth@google.com), Eric Anderson (ejona@google.com)
 * Approver: a11r
 * Status: Draft
-* Implemented in: (C-core and Java are in progress)
-* Last updated: 2017-01-18
+* Implemented in: C-core (Java is in progress)
+* Last updated: 2017-01-25
 * Discussion at: https://groups.google.com/d/topic/grpc-io/zG7iYpDw6lk/discussion
 
 ## Abstract
 
 Describes what types of TCP-level proxy setups gRPC will support and
-how those setups interact with load balancing.
+how those setups interact with per-call load balancing.
```

Comment:
> Since this document is mainly concerned with TCP-level proxies, I think it should be clear from context that that's not what's meant here.

It is not at all to me. If I didn't know that you made the change in response to my comment, I honestly would have no clue what you were talking about. If I had to guess, it would have been the wrong thing, but that's sort of expected if I have to guess.

How about you just say "GRPCLB" here? The other sentences you could say "client-side" without concern, since you're already calling out GRPC specifically.

> As I mentioned earlier, I dislike using the term "client-side" here, because grpclb is fundamentally not a client-side system -- it's a look-aside system.

I find that virtually every client-side LB system would be look-aside, because the client still needs to be told the list of addresses to use from some system. Say, from DNS or Zookeeper.

I've been using client-side LB to include GRPCLB, for several years, without apparent trouble. I thought that is what others call them. Google search doesn't seem to directly disagree. Lookaside LB turns up nothing on Google. Same for per-call LB.